server/webdriver_tests/screenshot/screenshot_test.py

```diff
@@ -14,11 +14,12 @@
 
 
 import time
-from base_test import WebdriverBaseTest
+from webdriver_tests.base_test import WebdriverBaseTest
 
+WIDTH = 1280
```

Comment:
feature request: we should run these tests at a few widths, at least 2 so we capture mobile browsers

Reply:
added a todo